src/common/obj_bencher.cc

```diff
@@ -638,13 +643,11 @@ int ObjBencher::seq_read_bench(int seconds_to_run, int num_objects, int concurre
   bufferlist* contents[concurrentios];
   int index[concurrentios];
   int errors = 0;
-  utime_t start_time;
-  std::vector<utime_t> start_times(concurrentios);
-  utime_t time_to_run;
-  time_to_run.set_from_double(seconds_to_run);
   double total_latency = 0;
   int r = 0;
-  utime_t runtime;
+  std::vector<mono_time> start_times(concurrentios);
```

Comment:
i am curious why you are reordering these variable definitions?

Reply:
That might have been inadvertent. I just like to group definition types and took the opportunity to do so. I'm happy to revert that if you want.

Reply:
yeah, makes sense to me! i just wanted to understand the rationale behind this change.